issue/github.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package issue
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/banzaicloud/pipeline/auth"
+	"github.com/google/go-github/github"
+	"github.com/spf13/viper"
+)
+
+// GitHubIssuer creates an issue on GitHub
+type GitHubIssuer struct {
+	Version VersionInformation
+}
+
+var _ Issuer = GitHubIssuer{}
+
+const issueTemplate = `
+**Version Information:**
```

Comment:
This could be formatted as a table.

Example:

```
| Q            | A
| ------------ | ---
| Bug?         | no|yes
| New Feature? | no|yes
| Version      | Specific version or SHA of a commit
```